src/model.ts

```diff
@@ -73,353 +83,225 @@ export class GitExtension implements IGitExtension {
   }
 
   /**
-   * The list of branch in the current repo
-   */
-  get branches() {
```

Comment:
Why did you move the getter/setter? We are following the JupyterLab style guide:

https://github.com/jupyterlab/jupyterlab/wiki/TypeScript-Style-Guide

> Start with getters/setters in each scope, then methods. Protected and private properties go last.

Reply:
Reverted.